include/crypto/aes_platform.h

```diff
@@ -75,6 +75,7 @@ void AES_xts_decrypt(const unsigned char *inp, unsigned char *out, size_t len,
 #   define HWAES_xts_encrypt aes_p8_xts_encrypt
 #   define HWAES_xts_decrypt aes_p8_xts_decrypt
 #   define PPC_AES_GCM_CAPABLE (OPENSSL_ppccap_P & PPC_MADD300)
+#  ifdef PPC_AES_GCM_CAPABLE
```

Comment:
This does not make much sense here. The define will be always defined. BTW, apparently the OPENSSL_ppccap_P & PPC_MADD300 is set because otherwise the stitched assembler implementation would not be called.